### DISCUSSION TITLE
减少直接使用sql语句，转向使用orm内置操作

### DISCUSSION THREAD

**Discussion by Mr-Ao-Dragon** [1Panel-dev/1Panel#5927] [Q&A]
### 1Panel 版本

v1.10.13-lts

### 请描述您的需求或者改进建议

翻看源码时，发现很多地方直接使用sql语句，这不仅给注入造成潜在的危险，也不符合orm一般使用习惯

### 请描述你建议的实现方案

寻找等义的orm操作代替直接的sql执行

**Comment by wanghe-fit2cloud** [comment]
项目使用的是 gorm 框架，近期针对 gorm 官方列出的潜在漏洞风险也进行了自查、发版。
